action.yml

```diff
@@ -0,0 +1,9 @@
+name: "leetcode-box"
+author: puiiyuen
+description: "Update a pinned gist to show your LeetCode stats"
+branding:
+  icon: code
+  color: blue
+runs:
+  using: "node12"

```

Comment:
changes the node version

Reply:
Transferred files from another repo and reviewed the PR before merging. I think that's a good habit for a developer @deployer117